armonik/variables.tf

```diff
@@ -471,3 +442,10 @@ variable "pod_deletion_cost" {
   })
   default = null
 }
+
+# Extra configuration
+variable "others_conf" {
+  description = "Variable"
+  type        = any
+  default     = {}
+}
```

Comment:
What is this? How is it different from `extra_conf`?

Reply:
Different type format. extra_conf is a map(string) and the others_conf is of the same type like the conf defined in the control_plane

Reply:
I am not sure about this. On one hand, I would prefer to avoid confusion of having two variables, and so keep only one. On the other hand, it would be good to not break entirely the compatibility.

That being said, the change is already really intrusive, so I would be good in breaking it a bit more.